pkg/dao/rpm.go

```diff
@@ -0,0 +1,171 @@
+package dao
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/content-services/content-sources-backend/pkg/api"
+	"github.com/content-services/content-sources-backend/pkg/models"
+	"gorm.io/gorm"
+)
+
+type rpmDaoImpl struct {
+	db *gorm.DB
+}
+
+func GetRpmDao(db *gorm.DB) RpmDao {
+	return rpmDaoImpl{
+		db: db,
+	}
+}
+
+func (r rpmDaoImpl) isOwnedRepository(orgID string, accountID string, repoUUID string) error {
+	var repoConfigs []models.RepositoryConfiguration
+	if err := r.db.
+		Where("org_id = ? and account_id = ? and repository_uuid = ?", orgID, accountID, repoUUID).
```

Comment:
i *Think* we should worry about org_id and not account id, so just query off of that